Cargo.toml

```
@@ -7,7 +7,9 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-serde_json = "1"
+serde_json = "1.0.91"
```

Prefer to leave as 1 really

```suggestion
serde_json = "1"
```